src/test/java/camp/nextstep/edu/calculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,79 @@
+package camp.nextstep.edu.calculator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+import java.util.stream.Stream;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class StringCalculatorTest {
+    @DisplayName(":를 구분자로 문자열을 파싱하여 문자열 숫자의 합을 반환한다")
+    @ParameterizedTest
+    @MethodSource("camp.nextstep.edu.calculator.StringCalculatorTestProvider#defaultStrategy")
```

Comment:
```suggestion
    @MethodSource("defaultStrategy")
```

테스트 코드 중복을 제거하기 위해 분리하신거라면 상속구조로 가져가는 것은 어떨까요